### ISSUE TITLE
width option to image directive causes latex error

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#38]
Adding a width option to an image directive breaks latex on my system (Ubuntu 8.04). These lines:

```
.. image:: ../flydra/autogenerated/plot_kalman_2d.png
  :width: 600
```

Cause the following output when "make all-pdf" is run.

```
<plot_kalman_2d.png, id=177, 578.16pt x 433.62pt>
! Illegal unit of measure (pt inserted).
<to be read again> 
                   \relax 
l.427 ...degraphics[width=600]{plot_kalman_2d.png}

? 
```

When the width option is removed, that PDF builds fine.

Thanks,
Andrew

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/38
- Originally reported by: [astraw](https://bitbucket.org/astraw)
- Originally created at: 2008-11-11T19:46:37.607

**Comment by shimizukawa** [comment]
_From [astraw](https://bitbucket.org/astraw) on 2008-11-11 18:49:45+00:00_

I should mention this is with <<cset f490645392fd4efbfbfa434f2d5d8bc1ebff2b7a>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2008-11-16 18:07:55+00:00_

Should be fixed in <<cset 49858b41f3a4ee4638ca8b8c1ee3ef332e4d5d82>>.
